nautobot/dcim/templates/dcim/location_migrate_data_to_contact.html

```diff
@@ -0,0 +1,131 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+
+{% block title %}Migrating contact data from {{ obj_type }} {{ obj }}{% endblock %}
+{% block form %}
+    <div id="assign" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Data</strong></div>
+            <div class="panel-body">
+                {% render_field form.action %}
+                <div class="form-group">
+                    <label class="col-md-3 control-label required">Source Location</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static">{{ obj.name }}</p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label required">Source Location Contact Name</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static">{{ obj.contact_name | placeholder}}</p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label required">Source Location Contact Phone</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static">{{ obj.contact_phone | placeholder}}</p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label required">Source Location Contact Email</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static">{{ obj.contact_email | placeholder}}</p>
+                    </div>
+                </div>
+                {% render_field form.name %}
+                {% render_field form.phone %}
+                {% render_field form.email %}
+                {% render_field form.contact %}
+                {% render_field form.team %}
+                {% render_field form.role %}
+                {% render_field form.status %}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block buttons %}
+    <button type="submit" name="_create" class="btn btn-primary">Assign</button>
+    <a href="{{ cancel_return_url }}" class="btn btn-default">Cancel</a>
```

Comment:
Can't this just be `{% url dcim:location pk=obj.pk %}`? Does it need to be part of the render context?